pttrack/urls.py

```diff
@@ -27,6 +27,9 @@
     url(r'^patient/update/(?P<pk>[0-9]+)$',
         views.PatientUpdate.as_view(),
         name='patient-update'),
+    url(r'^patient/activate/(?P<pk>[0-9]+)$',
+        views.patient_activate,
+        name='patient-activate'),
```

Comment:
I get that. I guess we can change it when we want since the urls are just pointing to content and we can control them.